server/src/main/java/io/deephaven/server/partitionedtable/PartitionedTableServiceGrpcImpl.java

```diff
@@ -161,6 +159,8 @@ public void getTable(GetTableRequest request, StreamObserver<ExportedTableCreati
                                         .get(requestedRow.getRowSet().firstRowKey());
                             });
                         }
+                        authWiring.checkPermissionGetTable(
+                                session.getAuthContext(), request, List.of(table, keys.get()));
```

Comment:
to confirm, this checkPermissionGetTable shouldn't validate the original PartitionedTable, but checks just the results? I don't _think_ the keys by themselves are necessarily helpful for this, except as an intermediate step?

Reply:
hmm.. @devinrsmith do you believe the permission handler would prefer a PartitionedTable or the result table? Can we propagate some sort of table attribute to the leaf nodes?